geopandas/tests/test_geodataframe.py

```diff
@@ -372,6 +368,20 @@ def test_transform(self):
         utm = lonlat.to_crs(epsg=26918)
         self.assertTrue(all(df2['geometry'].geom_almost_equals(utm['geometry'], decimal=2)))
 
+    def test_to_crs_geo_column_name(self):
+        """ Test to_crs() with different geometry column name (GH#339) """
+        df2 = self.df2.copy()
+        df2.crs = {'init': 'epsg:26918', 'no_defs': True}
+        cols = df2.columns.tolist()
+        cols[cols.index('geometry')] = 'geom'
+        df2.columns = cols
```

Comment:
``df2 = df2.rename(columns={'geometry':'geom'})`` is a bit cleaner for the three lines above